packages/teleterm/src/ui/QuickInput/QuickInput.story.tsx

```diff
@@ -28,6 +28,35 @@ export default {
 };
 
 export const Story = () => {
+  return (
+    <>
+      {/* Extra bottom margin to accommodate the suggestions displayed under the input. */}
+      <QuickInputDemo description="Pristine state" mb={350} />
```

Comment:
> The "scroll into viewport" mechanism in `QuickInput` causes the whole story to jump when you want to use an input at the bottom of the viewport.

Right, it's annoying but the reason I created the story in the first place is that I wanted to have an easy way to inspect all available suggestions.

But I suppose a better way would be to create a separate story which uses `QuickInputList` alone rather than setting up the whole `QuickInput` component. I'll make that change soon.